$DragonFly: src/gnu/usr.bin/cvs/cvs/Attic/cvs.1.patch,v 1.2 2006/09/13 19:34:24 corecode Exp $
diff -r 93f8116a881a contrib/cvs-1.12/doc/cvs.1
--- a/contrib/cvs-1.12/doc/cvs.1	Sat Sep 09 18:10:13 2006 +0200
+++ b/contrib/cvs-1.12/doc/cvs.1	Wed Sep 13 18:07:23 2006 +0200
@@ -420,6 +420,26 @@ set or \fB-r\fR is given.
 set or \fB-r\fR is given.
 .SP
 .IP "" 0
+\fB-g\fR
+.IP "" 2
+.IX "GroupPerms"
+Forces group-write perms on working files.  This option is typically
+used when you have multiple users sharing a single checked out source
+tree, allowing them to operate their shells with a less dangerous umask.
+To use this feature, create a directory to hold the checked-out source
+tree, set it to a private group, and set up the directory such that
+files created under it inherit the group id of the directory.  This occurs
+automatically with FreeBSD.  With SysV you must typically set the SGID bit
+on the directory.  The users who are to share the checked out tree must
+be placed in that group.  Note that the sharing of a single checked-out
+source tree is very different from giving several users access to a common
+CVS repository.  Access to a common CVS repository already maintains shared
+group-write perms and does not require this option.
+.SP
+To use the option transparently, simply place the line 'cvs -g' in your
+~/.cvsrc file.  Doing this is not recommended unless you firewall all your
+source checkouts within a private group or within a private mode 0700
+directory.
 \fB-x\fR
 .IP "" 2
 .IX "Encryption"
@@ -1079,13 +1099,20 @@ Changes: nothing.
 .SP
 For each file in \fIfiles\fR, print the head revision
 of the trunk, together with information on the last
-modification for each line.  
+modification for each line.  If \fB-b\fR is specified,
+the first modification after the revision selected by
+\fB-r\fR is shown.  
 .SP
 .SH "annotate options"
 .SP
 These standard options are supported by \fBannotate\fR
 (see node `Common options\(aq in the CVS manual, for a complete description of
 them):
+.SP
+.IP "" 0
+\fB-b\fR
+.IP "" 2
+Backwards, show when a line was removed.
 .SP
 .IP "" 0
 \fB-l\fR
